code/game/objects/items/crayons.dm

```diff
@@ -333,24 +334,31 @@
 	if(capped)
 		to_chat(user, "<span class='warning'>You cannot spray [target] while the cap is still on!</span>")
 		return
-	if(iscarbon(target))
-		if(uses - 10 > 0)
-			uses = uses - 10
-			var/mob/living/carbon/C = target
-			user.visible_message("<span class='danger'> [user] sprays [src] into the face of [target]!</span>")
-			if(C.client)
-				C.EyeBlurry(6 SECONDS)
-				C.EyeBlind(2 SECONDS)
-				if(ishuman(target))
-					var/mob/living/carbon/human/H = target
-					if(H.check_eye_prot() <= 0) // no eye protection? ARGH IT BURNS.
-						H.Confused(6 SECONDS)
-						H.KnockDown(6 SECONDS)
-					H.lip_style = "spray_face"
-					H.lip_color = colour
-					H.update_body()
-	playsound(user, 'sound/effects/spray.ogg', 5, TRUE, 5)
-	..()
+	if(istype(target, /obj/item/clothing/head/cardborg))	// Spraypainting your cardborg suit for more fashion options.
+		cardborg_recolor(target, user)
+		return
+	if(istype(target, /obj/item/clothing/suit/cardborg))
+		cardborg_recolor(target, user)
+		return
+	else
```

Comment:
Everything before this returns when TRUE, this else is not necessary.

Reply:
True...